packages/staking/src/features/store/types.ts

```diff
@@ -51,30 +51,41 @@ export type CurrentPortfolioStakePool = DraftPortfolioStakePool & {
   value: bigint;
 };
 
+export enum PortfolioState {
+  Free = 'Free',
+  ConfirmingNewPortfolio = 'ConfirmingNewPortfolio',
+  ManagingCurrentPortfolio = 'ManagingCurrentPortfolio',
+}
```

Comment:
What do you think about the following?

```suggestion
export enum PortfolioProcess {
  Idle = 'Idle',
  ManagingNewPortfolio = 'ManagingNewPortfolio',
  ManagingCurrentPortfolio = 'ManagingCurrentPortfolio',
}
```